Desktop/widgets/listmodelfiltereddataentry.cpp

```diff
@@ -155,6 +155,24 @@ void ListModelFilteredDataEntry::sourceTermsReset()
 	fillTable();
 }
 
+void ListModelFilteredDataEntry::initialValuesChanged()
+{
+	_initialValues.clear();
+	if (_tableView->initialValuesControl())
+	{
+		const Terms& terms = _tableView->initialValuesControl()->model()->terms();
+		if (terms.size() > 0)
+		{
+			std::string initColName = terms[0].asString();
+			DataSetTableModel* dataSetModel = DataSetTableModel::singleton();
+			int colIndex = dataSetModel->getColumnIndex(initColName);
+			for (int i = 0; i < dataSetModel->rowCount(); i++)
+				_initialValues.push_back(dataSetModel->data(dataSetModel->index(i, colIndex)).toDouble());
```

Comment:
Or actually it needs to be checked here if the value from the model is QVariant? Hmm

Reply:
_initialValues is used only in listModelFilteredDataEntry, so it should be anyway a double, and the toDouble just returns 0 if it cannot convert it. Or maybe I don't understand your remark...

Reply:
I meant more, that maybe it was going wrong because a model (DataSetModel in this case) return `QVariant()` if out of bounds. So I thought maybe initializing the initialValues maybe went wrong because the model was being used wrong or something. 

But it seems I just misunderstood the code which is fine. If the `defaultEmptyValue` is interpreted as variant and can be used I am totally happy.